Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM rsrchboy/perl-v5.16
+MAINTAINER Xinjiang Shao <shaoxinjiang@gmail.com>
+
+RUN apt-get update \
+    && apt-get install -y curl \
+    git \
+    libssl-dev \
+    libmpfr-dev \
+    nodejs \
+    npm \
+    nodejs-legacy \
+    && rm -fr /var/lib/apt/lists/*
+RUN npm install -g handlebars@1.3.0 uglifyjs
```

Comment:
Devs don't actually need this, should be able to remove node...

It's only necessary when you `dzil build` the bundle...